corehq/apps/email/models.py

```diff
@@ -0,0 +1,153 @@
+import jsonfield
+from django.db import models
+
+from corehq.apps.email.util import BadEmailConfigException, get_email_backend_classes
+from corehq.util.quickcache import quickcache
+
+
+class SQLEmailSMTPBackend(models.Model):
```

Comment:
If there is not going to be a couch version of this model, we could just drop `SQL` prefix.

Reply:
Changed